GroceryStores/GroceryStores/Pages/Index.cshtml.cs

```diff
@@ -20,67 +20,50 @@ public IndexModel(ILogger<IndexModel> logger)
-            {
-                string businessOwnerEndPoint = "https://data.cityofchicago.org/resource/r5kz-chrr.json";
-                string BusinessOwnerJsonString = webClient.DownloadString(businessOwnerEndPoint);
-                BusinessOwner[] allBusinessOwners = BusinessOwner.FromJson(BusinessOwnerJsonString);
-                ViewData["allBusinessOwners"] = allBusinessOwners;
+            //Business Owners data from the API
+            string BusinessOwnerJsonString = GetData("https://data.cityofchicago.org/resource/r5kz-chrr.json");
+            BusinessOwner[] allBusinessOwners = BusinessOwner.FromJson(BusinessOwnerJsonString);
+            ViewData["allBusinessOwners"] = allBusinessOwners;
 
-                string groceryStoreEndPoint = "https://data.cityofchicago.org/resource/53t8-wyrc.json";
-                string GroceryStoreJsonString = webClient.DownloadString(groceryStoreEndPoint);
-                GroceryStore[] allGroceryStores = GroceryStore.FromJson(GroceryStoreJsonString);
-                ViewData["allGroceryStores"] = allGroceryStores;
+            //Grocery stores data from te API
+            string GroceryStoreJsonString = GetData("https://data.cityofchicago.org/resource/53t8-wyrc.json");
+            GroceryStore[] allGroceryStores = GroceryStore.FromJson(GroceryStoreJsonString);
+            ViewData["allGroceryStores"] = allGroceryStores;
 
-                IDictionary<long, GroceryStore> groceryStoresMap = new Dictionary<long, GroceryStore>();
-                List<BusinessOwner> businessOwnerList = new List<BusinessOwner>();
+            //Map to store the the zipcode as key and grocery store as Value
+            IDictionary<long, GroceryStore> groceryStoresMap = new Dictionary<long, GroceryStore>();
 
-                foreach (GroceryStore grocery in allGroceryStores)
-                {
-                    if (!groceryStoresMap.ContainsKey(grocery.ZipCode))
-                    {
-                        groceryStoresMap.Add(grocery.ZipCode, grocery);
-                    }                    
-                }
+            //List to store filtered business owners
+            List<BusinessOwner> businessOwnerList = new List<BusinessOwner>();
 
 
-                foreach (BusinessOwner businessRec in allBusinessOwners)
+            //Iterating through Grocery stores and adding them to the groceryStoresMap
+            foreach (GroceryStore grocery in allGroceryStores)
+            {
+                if (!groceryStoresMap.ContainsKey(grocery.ZipCode))
                 {
-                    if (groceryStoresMap.ContainsKey(businessRec.ZipCode))
-                    {
-                        businessOwnerList.Add(businessRec);
-                    }
+                    groceryStoresMap.Add(grocery.ZipCode, grocery);
                 }
+            }
 
-                /*IDictionary<long, BusinessOwner> businessOwnersMap = new Dictionary<long, BusinessOwner>();
-
```

Comment:
Remove this entirely.  It's considered bad practice to commit commented-out code to version control, because you can always go back in history to see what used to be there.